src/EditorFeatures/CSharpTest/DocumentationComments/DocumentationCommentTests.cs

```diff
@@ -1114,6 +1114,156 @@ void M()
             VerifyPressingEnter(code, expected);
         }
 
+        [WorkItem(2108, "https://github.com/dotnet/roslyn/issues/2108")]
+        [Fact, Trait(Traits.Feature, Traits.Features.DocumentationComments)]
+        public void PressingEnter_Indentation1()
+        {
+            const string code =
+@"class C
+{
+    /// <summary>
+    ///     hello world$$
+    /// </summary>
+    void M()
+    {
+    }
+}";
+
+            const string expected =
+@"class C
+{
+    /// <summary>
+    ///     hello world
+    ///     $$
+    /// </summary>
+    void M()
+    {
+    }
+}";
+
+            VerifyPressingEnter(code, expected);
+        }
+
+        [WorkItem(2108, "https://github.com/dotnet/roslyn/issues/2108")]
+        [Fact, Trait(Traits.Feature, Traits.Features.DocumentationComments)]
+        public void PressingEnter_Indentation2()
+        {
+            const string code =
+@"class C
+{
+    /// <summary>
+    ///     hello $$world
+    /// </summary>
+    void M()
+    {
+    }
+}";
+
+            const string expected =
+@"class C
+{
+    /// <summary>
+    ///     hello 
+    ///     $$world
+    /// </summary>
+    void M()
+    {
+    }
+}";
+
+            VerifyPressingEnter(code, expected);
+        }
+
+        [WorkItem(2108, "https://github.com/dotnet/roslyn/issues/2108")]
+        [Fact, Trait(Traits.Feature, Traits.Features.DocumentationComments)]
+        public void PressingEnter_Indentation3()
+        {
+            const string code =
+@"class C
+{
+    /// <summary>
+    ///     hello$$ world
+    /// </summary>
+    void M()
+    {
+    }
+}";
+
+            const string expected =
+@"class C
+{
+    /// <summary>
+    ///     hello
+    ///     $$world
+    /// </summary>
+    void M()
+    {
+    }
+}";
+
+            VerifyPressingEnter(code, expected);
+        }
+
+        [WorkItem(2108, "https://github.com/dotnet/roslyn/issues/2108")]
+        [Fact, Trait(Traits.Feature, Traits.Features.DocumentationComments)]
+        public void PressingEnter_Indentation4()
+        {
+            const string code =
+@"class C
+{
+    /// <summary>
+    ///     $$hello world
+    /// </summary>
+    void M()
+    {
+    }
+}";
+
+            const string expected =
+@"class C
+{
+    /// <summary>
+    ///     
+    /// $$hello world
+    /// </summary>
+    void M()
+    {
+    }
+}";
+
+            VerifyPressingEnter(code, expected);
+        }
+
+        [WorkItem(2108, "https://github.com/dotnet/roslyn/issues/2108")]
+        [Fact, Trait(Traits.Feature, Traits.Features.DocumentationComments)]
+        public void PressingEnter_Indentation5_UseTabs()
+        {
+            const string code =
+@"class C
+{
+    /// <summary>
+	///     hello world$$
+    /// </summary>
+    void M()
+    {
+    }
+}";
+
+            const string expected =
+@"class C
+{
+    /// <summary>
+	///     hello world
+	///     $$
+    /// </summary>
+    void M()
+    {
+    }
+}";
+
```

Comment:
I'm not sure I understand precisely what you mean from your example. There are loads of tests in this file and I'm pretty sure those simple cases are already covered. Or am I missing something? (I usually am :smile:)

Note that this very test is checking tabs and spaces. useTabs is specified below to ensure that particular option is set and the test contains whitespace with tabs and spaces.